chatbot-api/requirements.txt

```diff
@@ -11,3 +11,97 @@ numpy==1.26.4
 pandas==2.2.2
 pgvector==0.2.5
 uvicorn==0.29.0
+Jinja2==3.1.3
```

Comment:
@shambhuhegde why are we adding all these additional dependencies besides `Jinja2`?

I trimmed down the deps to only the required deps, your change re-adds a lot of unused deps...